### ISSUE TITLE
Reported allele fraction precision

### ISSUE THREAD

**Issue by mbhall88** [jodyphelan/TBProfiler#257]
I have a few calls where the fraction is below 0.1, but is still called resistant. For example

```
Resistance variants report
-----------------
Genome Position Locus Tag       Gene    Variant Type    Change  Estimated Fraction      Drug    Annotation
761139  Rv0667  rpoB    missense_variant        p.His445Asp     1.000   Rifampicin
781687  Rv0682  rpsL    missense_variant        p.Lys43Arg      1.000   Streptomycin
2155168 Rv1908c katG    missense_variant        p.Ser315Asn     1.000   Isoniazid
2289090 Rv2043c pncA    missense_variant        p.His51Arg      0.088   Pyrazinamide
4247429 Rv3795  embB    missense_variant        p.Met306Val     1.000   Ethambutol
```

0.088 is less than 0.1, is there some weird float rounding going on here?

I am using v4.3.0

**Comment by jodyphelan** [comment]
Hi @mbhall88 

Sorry for the delay in getting back to you. I think this was fixed in v4.4.0. Could you check if that works?

**Comment by mbhall88** [comment]
Yep, all fixed in 4.4.0. Thanks!
